common/config/kafkaConfig.go

```diff
@@ -43,7 +43,8 @@ type (
 
 	// TopicConfig describes the mapping from topic to Kafka cluster
 	TopicConfig struct {
-		Cluster string `yaml:"cluster"`
+		Cluster  string `yaml:"cluster"`
+		IsSecure bool   `yaml:"isSecure,omitempty"`
```

Comment:
can you add comment on this field to explain what being secure means and what should user expect when it's set to true

Reply:
Sure, added comment. This will only affect our internal usage, OSS is using sarama producer which has been deprecated internally